OptimizelySDK/Config/DatafileProjectConfig.cs

```diff
@@ -541,6 +541,16 @@ public string GetAttributeId(string attributeKey)
             return null;
         }
 
+        /// <summary>
+        /// provides List of features associated with given experiment.
+        /// </summary>
+        /// <param name="experimentId">Experiment Id</param>
+        /// <returns>List| Feature flag ids list, null otherwise</returns>
+        public List<string> GetExperimentFeatureList(string experimentId)
```

Comment:
This method is invalid. How an experiment can be associated with multiple Features? Will discuss offline.

Reply:
Yes you are right but As you can see that ExperimentFeatureMap variable was already there I just made a method to return it, and it's returning list of features. So do you want me to update this existing dictionary as well?